doc/file-partition.md

```diff
@@ -0,0 +1,53 @@
+
+| Original Location       | New Location | Capacity Needs | i/o Frequency  |
+| ----------------------- | ------------ | -------------- | -------------- |
+| ${datadir}/blocks/index | ${datadir}   | low            | high           |
+| ${datadir}/blocks       | ${EXTERAL}   | high           | low            |
+| ${datadir}/chainstate   | n/a          | low            | high           |
+
+Note: the chainstate folder will not move if these instructions are followed closely.  Like the index folder, chainstate contains very high frequency LevelDB files and must remain on a fast (internal) disk if reasonable syncronization time is to be expected.
+
+Change "coinadm" in the following as appropriate for your Bitcoin administrative account.
+
+1) Start bitcoind with --datadir pointing to your internal drive:
+
+        bitcoind -daemon -conf=macos-bitcoin.conf -datadir=/Users/coinadm/local/bitcoin/data
+
+Let it run for a few minutes, or long enough that it has started syncho-
+   nizing.  Either tail the debug.log, or watch the blocks folder:
+   
+        ls -l /Users/coinadm/local/bitcoin/data/blocks/blk00000.dat
+        -rw-------  1 coinadm  staff  134191893 Jul 27 11:31 /Users/coinadm/local/data/blocks/blk00000.dat
+
+2) Stop bitcoind, so that we can rearrange some datadir folders:
+
+       kill -QUIT `cat /Volumes/WD-Passport-Mac/bitcoin/data/bitcoind.pid`
+
+3) Move the "index" folder up one level (so that it may remain on the internal drive). Like the 
+      
+          mv -f /Users/coinadm/local/bitcoin/data/blocks/index /Users/coinadm/local/bitcoin/data/
+
+4) Next, move the Bitcoin blockchain blocks folder off the internal drive:
+   
+          mkdir /Volumes/WD-Passport-Mac/bitcoin 
+          mv -f /Users/coinadm/local/bitcoin/data/blocks /Volumes/WD-Passport-Mac/bitcoin 
+
+Finally, set up soft links to restore the original folder structure:
+   
+| Folder Name              | Link Name           |
+| ------------------------ | ------------------- |
+| ${EXTERNAL}/blocks/index | ${datadir}/../index |
+| ${datadir}/blocks        | ${EXTERAL}/blocks   |
+
+5) Replace the original index folder location with a soft link:
+      
+          ln -s /Users/coinadm/local/bitcoin/index /Volumes/WD-Passport-Mac/bitcoin/blocks/index 
+
+6) Replace the original index folder location with a soft link:
+      
+          ln -s /Volumes/WD-Passport-Mac/bitcoin/blocks /Users/coinadm/local/bitcoin/data/blocks
+
+ 注意 - Nota - Note - ध्यान दें - ﻢﻠﺣﻮﻇﺓ - метка 
```

Comment:
Again, good question, thank you.  To me, the "key finding" here (if any, really) is simply that the "index" folder by default is nested *within* the "blocks" folder, unlike the "chainstate" folder which is a sibling folder of "blocks".  This *slightly* complicates moving the "blocks" folder off of the internal disk to an external disk; my apologies for repeating the obvious.  Database administrators out there (me included) might argue that this nested folder configuration is less desirable as it complicates physical separation of high-capacity/low-frequency block files from the lower-capacity/higher-frequency index (and chainstate) LevelDB files.
When I open the hood to my car there are warnings labels on the radiator cap, etc., etc.., and these labels are in multiple languages to point out appropriate cautions to naive vehicle operators who may have never looked under the hood or checked a radiator.  I'm taking inspiration from this and wanted to (hopefully) say "Note" in 1/2 dozen or so most common languages by usage.  Also, I want to be especially friendly in this day and age; I feel like we could all use it.
In closing, I hope it doesn't sound like I am trying to "make a mountain out of a molehill" here, it's not that at all; I just wanted to share my personal experiences in hopes of further facilitating ease of use of the system.  Overall I find the system to be very easy to work with and well thought out.